.github/workflows/ci.yml

```diff
@@ -3,34 +3,53 @@ name: CI
 on: [push]
 
 jobs:
-  build:
+  NodeCI:
 
     runs-on: ubuntu-latest
 
     steps:
     - uses: actions/checkout@v1
-    - name: Update Composer
-      run: sudo composer self-update
+
     - name: Cache vendor directory
       uses: actions/cache@preview
+      id: vendorcache
       with:
         path: vendor
-        key: ${{ runner.os }}-composer
+        key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
```

Comment:
Are you sure this path spec is what you want? Not `/composer.lock`?

Reply:
Same thing for `package-lock.json`

Reply:
It fails with `/composer.lock` and `composer.lock`, so I'm leaving it as-is.